WooCommerce/Classes/ViewRelated/Upgrades/WooPlanCardView.swift

```diff
@@ -0,0 +1,146 @@
+import Yosemite
+import SwiftUI
+
+struct WooPlanCardView: View {
+    let upgradePlan: WooWPComPlan
+    @Binding var selectedPlan: WooWPComPlan?
+
+    private var isSelected: Bool {
+        selectedPlan?.id == upgradePlan.id
+    }
+
+    private var isPopular: Bool {
+        let popularPlans =  [
```

Comment:
Is this popular plans logic used elsewhere? It seems like deciding which plans are popular needs to be in a more accessible/central place, such as the `InAppPurchasesForWPComPlansManager` class

Reply:
It is not used anywhere else. Since we only use it for that specific view due to design requirements I just moved it in the ViewModel that drives that view's data (https://github.com/woocommerce/woocommerce-ios/commit/2fe488acd833e42ecc880d72f6057e735a865612)